algorithms/facenet/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM python:3.6-slim-stretch
```

Comment:
Given that this container depends on tensorflow, do you want to inherit from one of the tensorflow base images instead (such as `tensorflow/tensorflow:1.10.0-py3`)? This would enable easy switching to GPU just by switching out the base image (e.g. to `tensorflow/tensorflow:1.10.0-gpu`).

Reply:
That image only has python 3.5 installed and you can't upgrade to python 3.6 because of the Ubuntu version. This causing breaking changes with the facenet repo because of variable hinting(https://www.python.org/dev/peps/pep-0526/). Here is the issue: https://github.com/tensorflow/tensorflow/issues/16875